test/parallel/test-crypto-prime.js

```diff
@@ -254,6 +255,18 @@ for (const checks of [-(2 ** 31), -1, 2 ** 31, 2 ** 32 - 1, 2 ** 32, 2 ** 50]) {
   });
 }
 
+{
+  const bytes = randomBytes(67108864);
```

Comment:
Is generating this large of a random number each time necessary? Should be possible to generate a value statically once that trips this error reliably.

Reply:
You mean storing the number into a file in fixtures and use that in the test?

Reply:
doesn't need to be in fixtures (can just be defined inline here) but yes that would work.

Reply:
I'll do it in a file: it's 64M.

Reply:
I think that's better than slowing down the CI every run by generating 64M random bytes.

Reply:
That said, couldn't it just be 64M of zeroes? It doesn't actually need to be random or a prime if I understand correctly.

Reply:
I don't think including a 64M file is going to fare better in the CI though, considering that it needs to download it especially whenever after we cleanup the workspace. If needed we can just put the test in pummel.

Reply:
Fortunately using a non-zero Buffer is enough, no need to generate a random number. PR updated, ptal. Thanks!